python/tk_houdini/ui_generation.py

```diff
@@ -134,6 +134,98 @@ def _itemNode(self, parent, label, id):
         node.text = id
         return item
 
+class AppCommandsPanels(AppCommandsUI):
+
```

Comment:
Added the doc string.

I agree about the name. I was trying to be consistent with `AppCommands`, `AppCommandsUI`, `AppCommandsMenu`, and `AppCommandsShelf`. But `AppCommandsPanels` is just silly. How about we compromise with `AppCommandsPanelHandler`? :)